### PR TITLE
Staking slashing

### PR DESCRIPTION
**Pull request by imyourm8**

### CODE REVIEW

**Comment by imyourm8** [comment]
I am working on more detailed slashing tests

File: contracts/staking/validatorShare/ValidatorShare.sol [hunk 1]
```diff
@@ -59,65 +59,67 @@ contract ValidatorShare is ValidatorShareStorage {
...
-        stakingLogger.logDelegatorClaimRewards(validatorId, msg.sender, liquidRewards, sharesToBurn);
+        if (liquidRewards >= minAmount) {
+            require(stakeManager.transferFunds(validatorId, liquidRewards, msg.sender), "Insufficent rewards");
+        }
+        return liquidRewards;
+    }
+
+    function withdrawRewards() public {
+        uint256 rewards = withdrawAndTransferRewards();
+        require(rewards >= minAmount, "Too small rewards amount");
```
Comment (0xAshish) [inline contracts/staking/validatorShare/ValidatorShare.sol:121]: This statement is duplicate.

Comment (imyourm8) [reply contracts/staking/validatorShare/ValidatorShare.sol:121]: Right, will fix

Comment (imyourm8) [reply contracts/staking/validatorShare/ValidatorShare.sol:121]: Hm, actually no, it's not a duplicate

**Pull request merged**